compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeBridge.desktop.kt

```diff
@@ -279,9 +279,43 @@ internal abstract class ComposeBridge(
         })
     }
 
+    // Decides which AWT events should be delivered, and which should be filtered out
+    private val awtEventFilter = object {
```

Comment:
Why do we need an extra object here? It seems like it can be just private functions

Reply:
We don't *need* it, but I think it's cleaner to bundle it into a "filter" object.